### PR TITLE
python3Packages.geomet: 0.2.1 -> 0.3.0

### PR DESCRIPTION
**Pull request by risicle**
###### Motivation for this change
Routine bump. Had to relax `cassandra-driver`'s version constraints to get it to work though.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/cassandra-driver/default.nix [hunk 1]
```diff
@@ -35,6 +35,10 @@ buildPythonPackage rec {
     sha256 = "1dn7iiavsrhh6i9hcyw0mk8j95r5ym0gbrvdca998hx2rnz5ark6";
   };
 
+  postPatch = ''
+    substituteInPlace setup.py --replace 'geomet>=0.1,<0.3' 'geomet'
```
Comment (turion) [inline pkgs/development/python-modules/cassandra-driver/default.nix:39]: Any chances of some version relaxation getting merged upstream?

File: pkgs/development/python-modules/geomet/default.nix [hunk 2]
```diff
@@ -8,24 +8,16 @@
...
   src = fetchFromGitHub {
     owner = "geomet";
     repo = "geomet";
     rev = version;
-    sha256 = "0fdi26glsmrsyqk86rnsfcqw79svn2b0ikdv89pq98ihrpwhn85y";
+    sha256 = "1lb0df78gkivsb7hy3ix0xccvcznvskip11hr5sgq5y76qnfc8p0";
   };
 
-  patches = [
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/geomet/default.nix:22]: Please remove the unused fetchpatch from inputs.

File: pkgs/development/python-modules/cassandra-driver/default.nix (hunk 1 above)
Comment (risicle) [reply pkgs/development/python-modules/cassandra-driver/default.nix:39]: Feel free to open an issue with them telling them all tests seem to pass with 0.3.

Comment (turion) [reply pkgs/development/python-modules/cassandra-driver/default.nix:39]: I did that in https://datastax-oss.atlassian.net/browse/PYTHON-1280, hopefully that's the right place.

**Review by turion — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 133432` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
4 packages built:
</summary>
<ul>
<li>python38Packages.cassandra-driver</li>
<li>python38Packages.geomet</li>
<li>python39Packages.cassandra-driver</li>
<li>python39Packages.geomet</li>
</ul>
</details>

**Pull request merged**